vowpalwabbit/slim/include/example_predict_builder.h

```diff
@@ -0,0 +1,22 @@
+#pragma once
+
+#include "vw_slim_predict.h"
+
+namespace vw_slim {
```

Comment:
please convert whitespace to spaces

Reply:
Fixed

---
In reply to: [270507016](https://github.com/VowpalWabbit/vowpal_wabbit/pull/1800#discussion_r270507016) [](ancestors = 270507016)